### PR TITLE
Add Paste

### PR DESCRIPTION
**Pull request by erdihu**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -710,6 +710,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
   * [localpaste](https://github.com/petermaloney/localpaste) - Simple python based pastebin you can run locally, with curl for input. Supports RAW text by default and binary data such as images (no extensions on returned url). `GPLv2` `Python`
   * [mojopaste](http://search.cpan.org/dist/App-mojopaste/) - Perl based pastebin. ([Demo](http://p.thorsen.pm/), [Source Code](https://github.com/jhthorsen/app-mojopaste)) `ARTv2` `Perl`
   * [NoteHub](https://notehub.org) - Free and Hassle-free Pastebin for Markdown Pages. Simple, clean, password provided, generated-short link. ([Demo](https://notehub.org), [Source Code](https://github.com/chmllr/NoteHub)) `MIT` `Nodejs`
+  * [Paste](https://bitbucket.org/j-samuel/paste/) - Paste is forked from the original source pastebin.com used before it was bought. ([Demo](http://phpaste.sourceforge.net/demo), [Source Code](https://bitbucket.org/j-samuel/paste/)) `GPLv3` `PHP`
```
Comment (n8225) [inline README.md]: Could you change the homepage to http://phpaste.sourceforge.net/  

> \* \[Paste\](http://phpaste.sourceforge.net/) - Paste is forked from the original source pastebin.com used before it was bought. \(\[Demo\]\(http://phpaste.sourceforge.net/demo), \[Source Code\]\(https://bitbucket.org/j-samuel/paste/)) \`GPLv3\` \`PHP\`

**Comment by erdihu** [comment]
Homepage changed to http://phpaste.sourceforge.net/

**Comment by jordansamuel** [comment]
Hi. Homepage for Paste is now https://phpaste.sourceforge.io/ and the repository has moved to GitHub: https://github.com/jordansamuel/PASTE

**Comment by nodiscc** [comment]
@jordansamuel can you edit the phpaste entry to reflect the changes? Thanks

**Pull request merged**